CHANGELOG.rst

```diff
@@ -1,3 +1,21 @@
+********************
+[0.2.2] - 2021-0X-XX
+********************
+
+**Bugfixes**:
+
+- Placement of mutations at non-inference sites was not always truely parsimonious,
+  since the ancestral state could not be pre-specified. This is fixed by using the
+  new map_mutations code from tskit 0.3.7 (:pr:`557`, :user:`hyanwong`)
+
+**New Features**:
+
+- Oldest nodes in a standard inferred tree sequence are no longer set to frequencies ~2
```

Comment:
This isn't a feature, it's a breaking change - people might be depending on the old behaviour.